tools/cpp/CMakeLists.txt

```diff
@@ -26,29 +26,35 @@ project (generate_geocoding_data)
 # Helper functions dealing with finding libraries and programs this library
 # depends on.
 include (gtest.cmake)
-include (FetchContent)
 
-# Downloading the abseil sources.
-FetchContent_Declare(
-    abseil-cpp
-    GIT_REPOSITORY  https://github.com/abseil/abseil-cpp.git
-    GIT_TAG         origin/master
-)
-
-# Building the abseil binaries
-FetchContent_GetProperties(abseil-cpp)
-if (NOT abseil-cpp_POPULATED)
-    FetchContent_Populate(abseil-cpp)
-endif ()
-
-if (NOT abseil-cpp_POPULATED)
-   message (FATAL_ERROR "Could not build abseil-cpp binaries.")
-endif ()
-
-# Safeguarding against any potential link errors as mentioned in
-# https://github.com/abseil/abseil-cpp/issues/225
-set(CMAKE_POSITION_INDEPENDENT_CODE TRUE)
-add_subdirectory(${abseil-cpp_SOURCE_DIR} ${abseil-cpp_BINARY_DIR})
+# Try using the existing absl
+find_package(absl REQUIRED)
```

Comment:
This needs to be
```cmake
find_package(absl)
```
in `cpp/CMakeLists.txt` (i.e. not this `CMakeLists.txt`).

We need to drop `REQUIRED` to avoid the error when `absl_FOUND` is false.

Reply:
As I am very less familiar with CPP, could you please send us a PR with necessary fixes?
That will help...

Reply:
The changes needed (against `master`) are in my comment here: https://github.com/google/libphonenumber/pull/2772#issuecomment-1184289820

But, sure, I can send a PR.

Reply:
New PR here: #2791